lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/passportCri/ContractTest.java

```diff
@@ -186,6 +186,27 @@ public RequestResponsePact validRequestReturnsValidAccessToken(PactDslWithProvid
                 .toPact();
     }
 
+    @Pact(provider = "PassportCriProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact invalidAccessTokenRequestReturnsError(PactDslWithProvider builder)
```

Comment:
Would be nice to say how this request is invalid (missing body)

Reply:
Oh, actually it looks like the idea is to use an invalid auth code, in which case the body needs to be in the request because that is where the auth code is used.